release/release_test.go

```diff
@@ -111,6 +111,8 @@ func (s *ReleaseTestSuite) TestForceDevMode(c *C) {
 		{id: "debian", devmode: true},
 		{id: "elementary", devmode: true},
 		{id: "elementary", idVersion: "0.4", devmode: false},
+		{id: "elementary OS", devmode: true},
+		{id: "elementary OS", idVersion: "0.4", devmode: false},
```

Comment:
Again, this is bogus. Let's not accept broken IDs.

Please use this logic for parsing the id out:
```
id = strings.Fields(strings.ToLower(id))[0]
```